styly.css

```diff
@@ -13,8 +13,31 @@ table {
     font-family: Arial;
 }
 
-td {
+thead th, tfoot th{
+    font-weight: bold;
+    border: solid #000000;
```

Comment:
neni to chyba, jen je zvykem uvadet sirku ramecku tj. napr.
```css
border: 1px solid #000000
```

ty jsi ji neuvedl, tak se pouzije hodnota, co ma prohlizec jako vychozi. Coz muze byt vsude jine (nebo nemusi ale ty nemas jistotu, ze to bude vsude vypadat jako u tebe)